woocommerce/woocommerce-gutenberg-products-block
# Move `Panel` to components package

## What

Moves `Panel` (used in the Cart/Checkout blocks) to the components package.

Fixes #11684

## Why

It doesn't make sense to have all of the components spread across three directories.



## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Install [Multiple Packages for WooCommerce](https://wordpress.org/plugins/multiple-packages-for-woocommerce/) and go to the settings for it (WooCommerce -> Settings) set the grouping option to Individual.
2. Ensure you have a few different shipping methods set up.
3. Add multiple items to your cart.
4. Go to the Cart block, ensure there are panels in the sidebar that can collapse/expand to show shipping options for each product.
<img width="376" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/c4c315ab-5fa5-4890-b0e5-f1caa7127404">

5. Go to the Checkout block. Shrink your screen to mobile size.
6. Ensure the "Order summary" panel appears and can be collapsed/expanded to show the items in your order.

| Closed | Open |
| ------- | ----- |
| <img width="470" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/8af080d9-f537-4296-8695-a9d7b89252ea"> | <img width="485" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/5656702/e4555d8f-d144-4b4d-b01c-ab4adb67416d"> |


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Move `Panel` component to the `@woocommerce/blocks-components` package.

## Dev note

We have moved the `Panel` component to the `@woocommerce/blocks-components` package. Previously this was available in `@woocommerce/blocks-checkout`. Your code will continue to work as we have added an alias to the new location. Despite this, it is recommended that you change your code to import this components from `@woocommerce/blocks-components` as the import from the checkout package will be deprecated in the future.
